extensions/authentication-jwt/package.json

```diff
@@ -47,6 +47,9 @@
     "lodash": "^4.17.21",
     "typescript": "~4.2.3"
   },
+  "optionalDependencies": {
+    "@node-rs/bcrypt": "^1.2.0"
```

Comment:
I would hesitate to list it as an optional dependency as binary addons cause too much trouble in `npm install`.

What about leaving it for applications to declare such dependency?

Reply:
`@node-rs/bcrypt` is not installed in the same way as other native addons, it doesn't contain any postinstall scripts/node-gyp things. Everything is pre-build correctly and publish as npm packages, so I think there will be no problem here.

[parser-server](https://github.com/parse-community/parse-server/blob/master/package.json#L134) landed `@node-rs/bcrypt` over 6 months and no feedback with installation problem for now.